packages/web-app/src/views/appeal-details.njk

```diff
@@ -0,0 +1,396 @@
+{% extends "layouts/form.njk" %}
```

Comment:
Doesn't look like this template needs to be a form so could extend `two-thirds-page`?